composer.json

```diff
@@ -51,6 +51,7 @@
     },
     "autoload-dev": {
         "classmap": [
+            "tests/concerns/InteractsWithAuthentication.php",
```

Comment:
Why does this have to be added to the composer file? Why can't it live with the rest of the test cases?

Reply:
@LukeTowers It's not a test case, it's a trait to enable emulating of an authenticated user. This was needed for the permissions tests. The reason for it being added there in Composer was because it is namespaced, but our tests folder in general is not currently namespaced.